src/analysis/retail/evoker/preservation/modules/talents/EssenceBurst.tsx

```diff
@@ -79,7 +79,7 @@ class EssenceBurst extends Analyzer {
     ].filter((item) => {
       return item.value > 0;
     });
-    return <DonutChart items={items} />;
+    return items.length > 0 ? <DonutChart items={items} /> : null;
```

Comment:
Small nit - I would rather see some indication that the talent is selected but no buffs were gained than just hiding the module entirely

Reply:
![image](https://user-images.githubusercontent.com/11250934/205535242-8238009f-8d5f-4c4a-b2c1-acc04a1fa797.png)
New statistic, gonna merge